integration/tests/form-behavior.spec.ts

```diff
@@ -70,6 +70,34 @@ test.describe('form behavior validation', () => {
 		expect(errorContainer).toContainText([`${label} is required.`]);
 	});
 
+	test('show inputGroupRequired message when radio group is missing a selection', async ({
```

Comment:
Can we add tests specifically around the fallback chain - i.e., that it uses the group label first?

Reply:
Adjusted.